lib/ext.js

```diff
@@ -100,7 +100,7 @@ module.exports = class Ext {
                 const name = this.name(arg);
                 let value = null;
 
-                if (request.query[name] || request.query[name] === 0) {
```

Comment:
@fknop I have removed the right side of this condition as I could not find a proper way to test it.
As far as I know, hapi will always pass the query parameters as a string, meaning `=== 0` will never be true.